{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}],"languages":["swift"],"platforms":["Mac Catalyst"]}]}],"identifier":{"url":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError\/unknown","interfaceLanguage":"swift"},"sections":[],"kind":"symbol","abstract":[{"type":"text","text":"Unknown error occured."}],"hierarchy":{"paths":[["doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication","doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError"]]},"variants":[{"paths":["\/documentation\/adyenauthentication\/adyenauthenticationerror\/unknown"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}],"title":"AdyenAuthenticationError.unknown","externalID":"s:19AdyenAuthentication0aB5ErrorO7unknownyA2CmF","modules":[{"name":"AdyenAuthentication"}],"symbolKind":"case","role":"symbol","roleHeading":"Case"},"schemaVersion":{"patch":0,"minor":3,"major":0},"references":{"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication/AdyenAuthenticationError":{"role":"symbol","identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError","abstract":[{"text":"An Error thrown by ","type":"text"},{"code":"AuthenticationService","type":"codeVoice"},{"text":".","type":"text"}],"title":"AdyenAuthenticationError","kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AdyenAuthenticationError","kind":"identifier"}],"type":"topic","navigatorTitle":[{"text":"AdyenAuthenticationError","kind":"identifier"}],"url":"\/documentation\/adyenauthentication\/adyenauthenticationerror"},"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication/AdyenAuthenticationError/unknown":{"title":"AdyenAuthenticationError.unknown","url":"\/documentation\/adyenauthentication\/adyenauthenticationerror\/unknown","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}],"identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication\/AdyenAuthenticationError\/unknown","type":"topic","role":"symbol","kind":"symbol","abstract":[{"text":"Unknown error occured.","type":"text"}]},"doc://com.adyen.AdyenAuthentication/documentation/AdyenAuthentication":{"abstract":[{"type":"text","text":"AdyenAuthentication SDK Provides reusable and easy to use two factor authentication"},{"type":"text","text":" "},{"type":"text","text":"for security sensitive use cases like banking, issuing and PSD2 strong customer authentication."}],"role":"collection","kind":"symbol","url":"\/documentation\/adyenauthentication","title":"AdyenAuthentication","identifier":"doc:\/\/com.adyen.AdyenAuthentication\/documentation\/AdyenAuthentication","type":"topic"}}}